js/src/get-started-page/index.js

```diff
@@ -14,6 +15,7 @@ const GetStartedPage = () => {
 			<GetStartedWithVideoCard />
 			<BenefitsCard />
 			<FeaturesCard />
+			<CaseStudiesCard />
```

Comment:
💅 

CaseStudies? In this case I guess it will be more accurate to use CustomerQuotes or Testimonials.

Reply:
Both terms of `case studies` and `customer quotes` appear in the design brief document in this post: pcTzPl-i7-p2#design-requirements. But I agree that `customer quotes` are more straightforward, updated in ca06c61, thanks!